src/App.js

```diff
@@ -10,9 +10,27 @@ import { getDayOfWeek, isValidDate } from "./utils/common";
 import Banner from "./components/banner/Banner";
 import { BannerAd } from "./components/ads/BannerAd";
 import { GoogleTags } from "./meta/GoogleTags";
+import { FilterBottomSheet, FilterMenuDesktop } from "./components/filters/FilterMenu";
+import { FilterIcon } from "./components/icons/Icons";
 
 export default function App() {
   const [searchParams, setSearchParams] = useSearchParams();
+  const [isOpen, setOpen] = useState(false);
+  const [numResults, setNumResults] = useState();
+  const [filters, setFilters] = useState({
```

Comment:
Not a blocker but the object here can be abstracted out so the app.js stays short in length.